pkgs/by-name/fr/frequest/package.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenv, fetchFromGitHub, qt5 }:
+
+let
+  version = "1.2a";
+
+  src = fetchFromGitHub {
+    owner = "fabiobento512";
+    name = "FRequest";
```

Comment:
```suggestion
```
those downloads default to src as a name which is good enough.

Reply:
Please see the comment above.